packages/react-components/react-nav-preview/src/components/NavCategory/NavCategory.tsx

```diff
@@ -1,22 +1,19 @@
 import * as React from 'react';
-import type { ForwardRefComponent } from '@fluentui/react-utilities';
-// import { useCustomStyleHook_unstable } from '@fluentui/react-shared-contexts';
 import { useNavCategory_unstable } from './useNavCategory';
 import { renderNavCategory_unstable } from './renderNavCategory';
-import { useNavCategoryStyles_unstable } from './useNavCategoryStyles.styles';
+import { useNavCategoryContextValues_unstable } from '../useNavCategoryContextValues_unstable';
+
+import type { ForwardRefComponent } from '@fluentui/react-utilities';
```

Comment:
Super nit: I put type imports before implementation imports when I can. I also put external package imports before local file imports. We considered some style rules, but in some rare cases imports have to be ordered precisely.

Reply:
It would be great if we had style guidelines about sorting imports. I use VS Code's Organize Imports (Shift+Alt+O), except always keeping the React import first (add an extra newline after the React import to prevent VS Code from sorting it in with the others).